api/app/Services/Leaderboard/Leaderboard.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Services\Leaderboard;
+
+use App\User;
+use App\Vote;
+
+class Leaderboard
+{
+    const CORRECT = 10;
+    const WRONG = -10;
+
+    const SKIP = 0;
+    const VOTE = 1;
+
+    public function forAllUsers()
```

Comment:
Nicely written but can't the computing power be reduced by adding a database column with the 'score' instead? 
I guess, the computing power can be distributed better in that way?